### ISSUE TITLE
type '_InternalLinkedHashMap<String, dynamic>' is not a subtype of type 'ParseBase'

### ISSUE THREAD

**Issue by SenhorBardell** [phillwiggins/flutter_parse_sdk#42]
# Steps to reproduce

- open example from repository
- run

# Description

It seems like on the request for health check, when response is returned it crashes on the `response.result = _handleSingleResult<T>(object, map);` line. I am not familiar enough with flutter and parse sdk to figure out the problem on my own yet.

**Comment by phillwiggins** [comment]
I'll check this out ASAP. Looks as though you Parse server URL might be
invalid but it should handle better than that.

**Comment by SenhorBardell** [comment]
Hey @phillwiggins that's for quick response.
My flutter version
```
Flutter 1.1.10-pre.78 • channel master • https://github.com/flutter/flutter.git
Framework • revision 105026cd68 (4 days ago) • 2019-01-16 08:13:00 -0800
Engine • revision 09e149e897
Tools • Dart 2.1.1 (build 2.1.1-dev.1.0 2195a70249)
```
I have this configuration inside `ApplicationConstantsts`
```abstract class ApplicationConstants {
  static const String keyAppName = "";
  static const String keyParseApplicationId = "test";
  static const String keyParseMasterKey = "key";
  static const String keyParseServerUrl = "http://localhost:1337/parse";
}```
and I run my parse-server with this command: `parse-server --appId test --masterKey key --databaseURI mongodb://localhost/project-name`

**Comment by phillwiggins** [comment]
Apologies, I meant the flutter_parse_sdk library being up to date.

It's strange as you are receiving a response from the server and that
should be handled but obviously is not matching what your server is sending
back.

Do you know how to debug the library... Make a breakpoint on the line
that's crashing. You should be able to get an idea of the response your
server is sending...

**Comment by erickvasilev** [comment]
![image](https://user-images.githubusercontent.com/18327978/51447018-22330600-1d4c-11e9-9b69-e8d1ecbc29cc.png)

I got this error after init:
var response = await Parse().healthCheck();

![image](https://user-images.githubusercontent.com/18327978/51447035-4db5f080-1d4c-11e9-972d-5552b6b01199.png)

parse_server_sdk: ^1.0.8

**Comment by cetorres** [comment]
Same thing here. It was working with the previous version of the lib. Now it fails on the first request to the Parse server, in my case, healthCheck() as well.
I cannot even start my app. Please fix it asap.
Thank you.

**Comment by phillwiggins** [comment]
Okay, I will look at this ASAP. Strange error...

For this release just remove healthcheck. It's not actually necessary, but
will be fixed in the next day or so.

**Comment by phillwiggins** [comment]
Sorry, just for reference, can you breakpoint before the exception. In really curious as to the contents of the response but won't be in from of a computer until this afternoon. Thanks in advance.

**Comment by cetorres** [comment]
It fails in any request. Healthcheck nust happened to be the first. But commented out and it fails on next request, a query.

**Comment by phillwiggins** [comment]
And Flutter Clean does nothing?

**Comment by erickvasilev** [comment]
Hi, not sure how to add a breakpoint in Flutter, but the server response is **{status: ok}**

![image](https://user-images.githubusercontent.com/18327978/51457985-15cb9f00-1d86-11e9-830d-11c7b5dc4c81.png)

**Comment by ScanEat** [comment]
I don't know if it helps you.

https://image.noelshack.com/fichiers/2019/04/1/1548062203-capture-d-ecran-2019-01-21-a-10-16-25.png

**Comment by ScanEat** [comment]
I don't know if it helps you :

https://image.noelshack.com/fichiers/2019/04/1/1548062409-capture-d-ecran-2019-01-21-a-10-16-25.png

**Comment by phillwiggins** [comment]
Perfect, I'll do some regression testing later and create an official 1.0.9 fix. Thanks for testing.

@ScanEat Do you want to create a pull request with your fixes? Or shall I add them to the release later?

**Comment by ScanEat** [comment]
Mmm, I prefer you do it because I don't feel comfortable with that.

**Comment by phillwiggins** [comment]
Fixed in 1.0.9
